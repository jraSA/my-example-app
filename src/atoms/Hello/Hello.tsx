import React, { ReactElement } from 'react'
import { Text } from 'ui-example-library';
import { myHelloWorldFunction } from 'core-example-library';

interface Props {
    name: string
}

export default function Hello({ name }: Props): ReactElement {
    return (
        <Text>{myHelloWorldFunction(name)}</Text>
    );
}
