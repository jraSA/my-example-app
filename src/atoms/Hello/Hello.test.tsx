import React from 'react';
import { render } from '@testing-library/react';
import Hello from '.';

it('renders correctly', () => {
    const tree = render(
        <Hello name={'foo-boo'} />
    );
    expect(tree.container).toMatchSnapshot();
});