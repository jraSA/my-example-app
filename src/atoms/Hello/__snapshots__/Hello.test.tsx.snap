// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    class="sc-bdfBwQ jDXXXp"
  >
    Hello foo-boo!!
  </div>
</div>
`;
